java/client/src/main/java/glide/api/BaseClient.java

```diff
@@ -65,6 +65,60 @@ protected static String handleStringResponse(Response response) {
                         + " expected String");
     }
 
+    /**
+     * Extracts the response value from the Redis response and either throws an exception or returns
+     * the value as a Long.
+     *
+     * @param response Redis protobuf message
+     * @return Response as a Long
+     */
+    public static Long handleLongResponse(Response response) {
+        Object value = handleObjectResponse(response);
+        if (value instanceof Long) {
```

Comment:
`Long` or `long`?

Reply:
Long that's the value returned from lib.rs and I don't think compleatable future generics allow for primitive types.